diff --git a/build0_win.xsh b/build0_win.xsh
index 8530d835b..b8bba80f4 100644
--- a/build0_win.xsh
+++ b/build0_win.xsh
@@ -12,6 +12,8 @@ python src/libasr/asdl_cpp.py grammar/Python.asdl src/lpython/python_ast.h
 python src/libasr/asdl_cpp.py src/libasr/ASR.asdl src/libasr/asr.h
 # Generate a wasm_visitor.h from src/libasr/wasm_instructions.txt (C++)
 python src/libasr/wasm_instructions_visitor.py
+# Generate the intrinsic_function_registry_util.h (C++)
+python src/libasr/intrinsic_func_registry_util_gen.py
 
 # Generate the tokenizer and parser
 pushd src/lpython/parser && re2c -W -b tokenizer.re -o tokenizer.cpp && popd
